serialization/journal.proto

```diff
@@ -1478,6 +1478,32 @@ message SetBufferedLogging {
   optional In in = 1;
 }
 
+message SetEphemerisParameters {
+  extend Method {
+    optional SetEphemerisParameters extension = 5148;
+  }
+  message In {
+    required fixed64 plugin = 1 [(pointer_to) = "Plugin", (is_subject) = true];
+    // Must be the name of one of the values of FixedStepSizeIntegrator.Kind.
+    required string fixed_step_size_integrator_kind = 2;
```

Comment:
Let's move these two parameters to a message `FixedStepConfigurationParameters` near the beginning of this file and use it here and in the next entry point.

Reply:
Done.